src/components/highlight-area.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default React.forwardRef(function HighlightArea(props, ref) {
+  const { startPoint, endPoint } = props;
+  const width = endPoint.x - startPoint.x;
+  const height = endPoint.y - startPoint.y;
+
+  return (
+    <rect
+      width={Math.abs(width)}
+      height={Math.abs(height)}
+      ref={ref}
+      x={width > 0 ? startPoint.x : endPoint.x}
+      y={height > 0 ? startPoint.y : endPoint.y}
+      style={{
+        fill: 'rgb(0, 149, 210)',
```

Comment:
Maybe add className so the selection rectangle could be updated if people want to? (Assume this will work for most scenarios now, so it's not necessary to expose as color prop variable)

Reply:
Thanks. Added a class name.